crates/gossip/proto/gossip.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package kitsune2.gossip;
+
+// A Kitsune2 gossip protocol message.
+message K2GossipProto {
+  oneof gossip_message {
+    // A gossip initiation protocol message.
+    K2GossipInitiateProto initiate = 1;
+
+    // A gossip acceptance protocol message.
+    K2GossipAcceptProto accept = 2;
+  }
+}
+
+// A Kitsune2 gossip initiation protocol message.
+message K2GossipInitiateProto {
+  // The agents from the initiator who are participating in this gossip round.
+  repeated bytes participating_agents = 1;
```

Comment:
Are these full agent infos? Perhaps add some docs about the contents here.

Reply:
They are not, they are just the ids. I believe this is similar to what we currently do in gossip, send the ids, check against the store, request what was missing.

Updated the doc comments to make this clearer